juju/juju
# Fixed model listing for admin user.

Admin user needs to see all models when listing,
list models is not the nicest method and does a direct
query to mongo which returns a partial set of models
owned by the calling user.

## QA steps
* Bootstrap juju
* create a user and grant add-model to said user
* login with the user and create a model
* create a second user.
* grant write to the second user on the model owned by the first user.
* all should work.

## Bug reference

https://bugs.launchpad.net/juju/+bug/1650643
